test/pet-store-swagger-inheritance.yaml

```diff
@@ -170,7 +170,7 @@ definitions:
       properties:
         packSize:
           type: integer
-          format: int32
+          # format: int32
```

Comment:
why is this comment out?

Reply:
you right no need to be comment, I remove this and all the swagger that were change